server/api.go

```diff
@@ -148,13 +150,14 @@ func (p *Plugin) submitScore(w http.ResponseWriter, r *http.Request) {
 	}
 
 	var score int
-	if i, err := getScore(surveyResponse.Context["selected_option"].(string)); err != nil {
+	var i int64
+	var err error
+	if i, err = getScore(surveyResponse.Context["selected_option"].(string)); err != nil {
```

Comment:
Does moving the `getScore` call out of the `if` also fixes the linter error?

Reply:
yes, because I the linter say the errors *if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary)*.

Reply:
Given that you already dropped the previous `else`, could you double check if that is still the case?

Reply:
yes, and the result it's work and all linter is pass